### DISCUSSION TITLE
Is there an option to auto-select the first item in the search results?

### DISCUSSION THREAD

**Discussion by andikrueger** [microsoft-search/pnp-modern-search#2566] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm building a search based solution that uses two search webparts. The first one shows a list of items that a selectable and the second one should show details based on the selection within the first webpart. Is there any option to always pre-select the first item in the first webpart, so that the second webpart automatically holds some content?

**Describe the solution you'd like**
Would be great to have an additional option within the webpart to force the selection of the first item in the webpart.

**Describe alternatives you've considered**
-

**Additional context**
-

**Comment by FranckyC** [comment]
Hi @andikrueger, I'll tag it as enhancement as it is not possible as of today.
